client/js/options.js

```diff
@@ -2,15 +2,29 @@
 
 const $ = require("jquery");
 const escapeRegExp = require("lodash/escapeRegExp");
-const userStyles = $("#user-specified-css");
 const storage = require("./localStorage");
 const tz = require("./libs/handlebars/tz");
+const socket = require("./socket");
 
-const windows = $("#windows");
-const chat = $("#chat");
+const $windows = $("#windows");
+const $chat = $("#chat");
+const $settings = $("#settings");
+const $theme = $("#theme");
+const $userStyles = $("#user-specified-css");
 
-// Default options
-const options = {
+const noCSSparamReg = /[?&]nocss/;
+
+// Not yet available at this point but used in various functionaly.
+// Will be assigned when `initialize` is called.
+let $syncWarningOverride;
+let $syncWarningBase;
```

Comment:
I don’t think we prefix element objects anywhere else with $

Reply:
It is a good habit to do it with jqueury objects, makes it easier to read code.